mysql-test/suite/rocksdb/r/rocksdb.result

```diff
@@ -1595,6 +1615,16 @@ ROCKSDB_WRITE_WAL
 #  but they are shown as both session and global, like InnoDB's status vars.
 select VARIABLE_NAME from INFORMATION_SCHEMA.session_status where VARIABLE_NAME LIKE 'rocksdb%';
```

Comment:
If there are non-deterministic results from this query, you can add an order by clause to this select.